services/web/client/source/class/osparc/dashboard/StudyBrowserListBase.js

```diff
@@ -0,0 +1,181 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2018 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+     * Tobias Oetiker (oetiker)
+
+************************************************************************ */
+
+/**
+ * Widget used mainly by StudyBrowser for displaying Studies
+ *
+ * It consists of a thumbnail and creator and last change as caption
+ */
+
+qx.Class.define("osparc.dashboard.StudyBrowserListBase", {
```

Comment:
The name it is a bit confusing since it is a button, extends a button and the name suggests it is a list
Maybe StudyBrowserButtonBase?

Reply:
Also maybe it would be cool to just reuse the same widget for the new buttons as well but not so important...